libs/form-configuration/core/src/loader/fetch-form-configurations.ts

```diff
@@ -19,6 +19,13 @@ import { FormConfiguration, FormConfigurationConfiguration, FormYupConfiguration
 import { FormConfigurationValidationConverter } from "../converter";
 import { useFormConfigurationStore } from "../store";
 
+const mergeFormYupConfigurationsWithoutDuplicates = (oldFormYupConfiguration: FormYupConfiguration[], newFormYupConfiguration: FormYupConfiguration[]) => {
+  const mergedFormYupConfigurations = [...oldFormYupConfiguration, ...newFormYupConfiguration];
+
+  // next line removes all duplicates based on formId
+  return [...new Map(mergedFormYupConfigurations.map((item) => [item.formId, item])).values()];
```

Comment:
i think we have lodash on this module. can you maybe find a lodash function that does this? Currenlty this looks hacky :D